code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -85,7 +85,9 @@
 	return
 
 /datum/antagonist/traitor/human/forge_traitor_objectives()
-	var/is_hijacker = prob(10)
+	var/is_hijacker = FALSE
+	if (GLOB.joined_player_list.len > 30) // Less murderboning on lowpop thanks
```

Comment:
also this should probably be `>=`

Reply:
PR title is more than 30, but I can make it 30 and more, give me a second.